.pipelines/templates/step/finalize.yml

```diff
@@ -0,0 +1,7 @@
+# This Yaml Document has been converted by ESAI Yaml Pipeline Conversion Tool.
+# Please make sure to check all the converted content, it is your team's responsibility to make sure that the pipeline is still valid and functions as expected.
+steps:
+- pwsh: |
```

Comment:
This may not be needed in OneBranch

Reply:
I will remove this with packaging changes